alerts/alerts.libsonnet

```diff
@@ -0,0 +1,38 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'gluster-utilization',
+        rules: [
+          {
+            alert: 'GlusterCpuPercentage',
+            expr: |||
+                  gluster_cpu_percentage > 0.8,
+            |||,
+            'for': '1m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Gluster - CPU Utilization more than 80%',
```

Comment:
Is this generic format to attach `Gluster - ` in the start for all the alert messages?

Reply:
Not really.
From what I have seen, usually the combination of `summary` `description` and `runbook` is used with `annotations` to describe any alert that ocurs.

Since these details aren't available yet, I'm using this format to distinguish it from the K8s alerts.